GameCenterManager.xcodeproj/project.pbxproj

```diff
@@ -572,10 +564,6 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				CLANG_ENABLE_OBJC_ARC = YES;
 				CLANG_WARN_EMPTY_BODY = YES;
-				CODE_SIGN_ENTITLEMENTS = "GameCenterManager Mac/GameCenterManager Mac.entitlements";
```

Comment:
Fixed up Coding signing issues. Now automatically assigned... not set to iRare Media's device / signing